roles/sensu-check/defaults/main.yml

```diff
@@ -10,28 +10,37 @@
 
 sensu_checks:
   apache:
-    check_apache_process:
+    check_apache_process_ubuntu:
       handler: default
       notification: "unexpected number of apache2 processes"
-      interval: 120
+      interval: 30
       standalone: true
-      command: "check-procs.rb -p apache2 -w 50 -c 100 -W 1 -C 1"
+      command: "/etc/sensu/plugins/check-procs.rb -p apache2 -w 50 -c 100 -W 1 -C 1"
       service_owner: openstack
-
+      os: ubuntu
+    check_apache_process_rhel:
+      handler: default
+      notification: "unexpected number of httpd processes"
+      interval: 30
+      standalone: true
+      command: "/etc/sensu/plugins/check-procs.rb -p httpd -w 50 -c 100 -W 1 -C 1"
+      service_owner: openstack
+      os: rhel
+      
   common:
     check_sshd_process:
       handler: default
       notification: "unexpected number of sshd processes"
       interval: 120
       standalone: true
-      command: "check-procs.rb -p sshd -w 50 -c 100 -W 1 -C 1"
+      command: "/etc/sensu/plugins/check-procs.rb -p sshd -w 50 -c 100 -W 1 -C 1"
```

Comment:
why is this path needed ?  I think we do or can set a path in the sensu config specifically to avoid needing to do this.